interop-testing/src/test/java/io/grpc/testing/integration/ProxyTest.java

```diff
@@ -202,7 +202,7 @@ public void run() {
   }
 
   // server with echo and streaming modes
-  private class Server implements Runnable {
+  private static class Server implements Runnable {
```

Comment:
I don't think that relates at all to using errorprone or adding static here.

I actually prefer it the way it is. Yes, I could add final, but since it is a private class already I don't see that as doing much. Concerning the visibility, I tend to prefer visibility be "normal" inside the class and only control the class's visibility to choose whether to expose it to others. This makes them a bit easier to read and also easier to move. So I sort of prefer the way this is already done.